src/backend/app/main.py

```diff
@@ -1,13 +1,18 @@
-from fastapi import FastAPI, Request, Depends
+import os
+from copy import deepcopy
+from datetime import datetime
+
+from fastapi import Depends, FastAPI, Request
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.responses import JSONResponse
-from datetime import datetime
 from sqlalchemy.orm import Session
-import os
 
 from . import tools
 from .db import models
-from .db.database import engine, SessionLocal, check_db_connection
+from .db.database import SessionLocal, check_db_connection, engine
```

Comment:
Thanks for reorganizing the imports! It's a minor thing, but it does make it easier to scan the dependencies.